docs/modules/ROOT/pages/general-info/general-info.adoc

```diff
@@ -1,138 +1,107 @@
 
-_Since Payara Server 4.1.152_
+[cols=-2*,options="header"]
+|=== 
 
-We currently have a set strategy to balance receiving the latest Glassfish
-upstream changes with a need to avoiding making late changes to a release.
+| Release date| Version  
 
-[[change-freeze]]
-=== Change Freeze
+| June 2020 | 5.20.0
 
-Two to three weeks before a release, we will implement a change freeze.
-This means that no new features or changes will go into the coming release
-of Payara Server.
+| July 2020 | 5.20.1
 
-The remaining two to three weeks will be used for finding any bugs brought
-on by changes and fixes implemented for the coming release. If any bugs are
-found, these will be corrected before the release; if we can't fix it before
-the release date, we will simply revert the change that caused the bug
-(pending a review).
+| August 2020 | 5.20.2
 
-[[glassfish-upstream-cherry-pick]]
-=== GlassFish Upstream Cherry-Pick
+| September 2020 | 5.21.0
 
-We aim to do three cherry picks from the GlassFish upstream for each
-release: just after a release; at the beginning of the penultimate month
-of the release; and again at the beginning of the final month of the
-release (before the change freeze). + This to allow us time to find and
-fix any errors that may be brought in from the GlassFish upstream.
+| ... | 5.21.1
 
-If a bug fix is committed to the GlassFish upstream during our change freeze, we will consider pulling in that solitary commit before the release.
+| ...| ...
 
-[[github]]
-== GitHub
+| March 2021 | 5.23.0
 
-The Payara project is hosted on GitHub, allowing us to make use of the
-various tools GitHub has integrated for managing projects.
+| April 2021 | 5.23.1
 
-GitHub also makes allowing the community access to view and edit the source
-code very easy and public.
+| May 2021 | 5.23.2
 
-[[how-to-contribute]]
-=== How to Contribute
+| June 2021 | 5.24.0
 
-We make use of  https://help.github.com/articles/using-pull-requests/[the Fork and Pull model].
-This means that if you want to make any changes, you'll need to fork the
-Payara project and make your changes in your own repository. You will then
-need to create a pull request back into the Payara project's master branch
-to merge your changes into the main project.
+| ... | 5.24.1
 
-Before we merge your pull request though, you must read and sign the
-Individual Contributor License Agreement (CLA) before sending it to us at
-the address specified on the document, or as an email attachment to
-cla@payara.org.
+| ...| ...
 
-Once we've received the CLA and checked it for any mistakes, we'll allow
-any pull requests you've made to start being merged.
+|=== 
 
-In most cases, pull requests will not be merged immediately. This is to
-allow the Payara team, and other members of the community, to review and
-deliberate over any of the changes made; we will typically wait 2-3 days
-before merging any pull requests.
+[[naming-strategy]]
```

Comment:
This is a naming strategy for Payara 4, is it relevant? I would describe the previous naming strategy for Payara 5, e.g. 5.201, 5.194, ...

Reply:
This is already on the documentation page today in this format. We can change this with another PR unrelated to split Community / Enterprise.